wallaby.js

```
@@ -0,0 +1,111 @@
+module.exports = function () {
```

ESLint

[error] 'module' is not defined.

(no-undef)